xml/System.Security.Authentication/HashAlgorithmType.xml

```diff
@@ -76,7 +76,7 @@
       </ReturnValue>
       <MemberValue>32771</MemberValue>
       <Docs>
-        <summary>The Message Digest 5 (MD5) hashing algorithm.</summary>
+        <summary>The Message Digest 5 (MD5) hashing algorithm. Due to collision problems with MD5, Microsoft recommends SHA256.</summary>
```

Comment:
These two changes in this file should also move to `remarks`.

I think after this change it should be good to merge. Thanks again!